translations/monero-core_fr.ts

```diff
@@ -1079,235 +1079,242 @@ The following information will be deleted
 
 The old wallet cache file will be renamed and can be restored later.
 </source>
-        <translation type="unfinished"></translation>
+        <translation>Êtes-vous sûr de vouloir reconstruire le cache du portefeuille ?
+Les informations suivantes seront supprimées
+- Adresses de destinaires
+- Clefs de transactions
+- Descriptions des transactions
+
+L'ancien fichier du cache du portefeuille sera renommé et pourra être restauré ultérieurement.
+</translation>
     </message>
     <message>
         <location filename="../pages/settings/SettingsInfo.qml" line="188"/>
         <source>Cancel</source>
-        <translation type="unfinished">Annuler</translation>
+        <translation>Annuler</translation>
     </message>
     <message>
         <location filename="../pages/settings/SettingsInfo.qml" line="202"/>
         <source>Invalid restore height specified. Must be a number.</source>
-        <translation type="unfinished"></translation>
+        <translation>Hauteur de restauration spécifiée invalide. Doit être un nombre.</translation>
     </message>
     <message>
         <location filename="../pages/settings/SettingsInfo.qml" line="236"/>
         <source>Wallet log path: </source>
-        <translation type="unfinished"></translation>
+        <translation>Chemin des journaux du portefeuille :</translation>
     </message>
     <message>
         <location filename="../pages/settings/SettingsInfo.qml" line="267"/>
         <source>Copy to clipboard</source>
-        <translation type="unfinished"></translation>
+        <translation>Copier dans le presse-papiers</translation>
     </message>
     <message>
         <location filename="../pages/settings/SettingsInfo.qml" line="292"/>
         <source>Copied to clipboard</source>
-        <translation type="unfinished">Copié dans le presse-papier</translation>
+        <translation>Copié dans le presse-papier</translation>
     </message>
 </context>
 <context>
     <name>SettingsLayout</name>
     <message>
         <location filename="../pages/settings/SettingsLayout.qml" line="59"/>
         <source>Custom decorations</source>
-        <translation type="unfinished"></translation>
+        <translation>Décor personnalisé</translation>
     </message>
     <message>
         <location filename="../pages/settings/SettingsLayout.qml" line="67"/>
         <source>No Layout options exist yet in mobile mode.</source>
-        <translation type="unfinished"></translation>
+        <translation>Aucune option d'agencement n'est disponible pour le moment en mode mobile.</translation>
```

Comment:
layout is translated by "disposition" on line 705 - probably better to be consistent

Reply:
I keep "agencement" and change L705 then.